ow_regolith/include/RegolithSpawner.h

```diff
@@ -0,0 +1,82 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef REGOLITH_SPAWNER_H
+#define REGOLITH_SPAWNER_H
+
+#include <ros/ros.h>
+#include <tf/transform_datatypes.h>
+
+#include "ow_dynamic_terrain/modified_terrain_diff.h"
+#include "ow_lander/DigLinearActionResult.h"
+#include "ow_lander/DigCircularActionResult.h"
+#include "ow_lander/DeliverActionResult.h"
+
+class RegolithSpawner
+{
+public:
+  RegolithSpawner() = delete;
```

Comment:
Good.  I would also be explicit regarding the copy constructor, assignment operator, and destructor.

Reply:
Agreed. It's good to be thorough. Could you explain the reasoning for deleting the class' destructor? Is this simply done if you never intend to manually allocate/deallocate a class using new/delete keywords?

Reply:
Sorry, I misunderstood. All the methods you mentioned are now explicitly defined or deleted.